ftfy/__init__.py

```diff
@@ -195,6 +193,7 @@ def fix_text(text,
 
 def fix_file(input_file,
              encoding=None,
+             *,
```

Comment:
To make the set of keyword-only arguments the same as `fix_text`. And it's reasonable for someone to have given the encoding as a positional argument.